docs/concepts/fencing.md

```diff
@@ -1,98 +1,48 @@
+- If one pod is terminating, another will be automatically scheduled to start even if the first pod is not yet completely terminated. Stateless applications benefit from this behaviour as one pod executes the same work as any other in the deployment.
+
+StatefulSets, on the other hand, **guarantee that every pod scheduled has a unique identity**, which is to say that only a single copy of a pod is running in the cluster at any one time. 
+- Whenever scheduling decisions are made, the StatefulSet controller ensures that only one copy of this pod is running at any time. 
+- If a pod is deleted, a new pod will not be scheduled until the first pod is fully terminated. This is an important guarantee as file systems need to be unmounted before they can be remounted in a new pod. Any [ReadWriteOnce (RWO)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) PVC defining a device requires this behaviour to ensure the consistency of the data and thus the PVC.
+
+To protect data integrity, Kubernetes guarantees that there will never be more than one instance of a StatefulSet pod running at a time. It assumes that when a node is determined to be offline it may still be running the workload but partitioned from the network. Since Kubernetes is unable to verify that the pod has been stopped it errors on the side of caution and does not allow a replacement to start on another node.
+
+Kubernetes does reschedule pods from some controllers when nodes become unavailable. The default behaviour is that when a node becomes unavailable its status becomes `Unknown` and after the `pod-eviction-timeout` has passed pods are scheduled for deletion. By default, the `pod-eviction-timeout` is `300` seconds.
+- For this reason, Kubernetes requires manual intervention to initiate timely failover of a StatefulSet pod. The **Ondat Fencing Controller** gives the capability to enable fast failover for workloads when a node goes offline.
+
+For more information on the rationale behind the design of StatefulSets, review the Kubernetes design proposal archive for [Pod Safety, Consistency Guarantees, and Storage Implications](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/pod-safety.md).
+
+### Ondat Fencing Controller
+
+> 💡 The Ondat Fencing Controller is part of the Ondat API Manager which is deployed in high availability mode when Ondat is installed.
+
+> 💡 High Availability for StatefulSet applications can be achieved with the Ondat Fencing feature.
+
+Since Ondat is able to determine when a node is no longer able to access a volume and has protections in place to ensure that a partitioned or formerly partitioned node can not continue to write data, it can work with Kubernetes to perform safe, fast failovers of pods, including those running in StatefulSets.
```

Comment:
"can stop writing data" instead of "can not continue to write data"?

Reply:
fixed